sources/zsh/history.d/tailscale

```diff
@@ -0,0 +1,2 @@
+tailscale up --advertise-routes=10.10.11.0/24
+tailscaled --tun=userspace-networking --socks5-server=localhost:1055 &>/dev/null &
```

Comment:
Missing new line here too at the end of the last line

Reply:
Should be okay as-is, we have a function that cleans things up